plugins/base/src/main/kotlin/translators/psi/JavadocParser.kt

```diff
@@ -119,88 +131,118 @@ class JavadocParser(
         }
     }
 
-    private fun getSeeTagElementContent(tag: PsiDocTag): List<DocTag> =
-        listOfNotNull(tag.referenceElement()?.toDocumentationLink())
+    private fun getSeeTagElementContent(tag: PsiDocTag): Pair<List<DocumentationLink>, DRI?> {
+        val content = tag.referenceElement()?.toDocumentationLink()
+        return Pair(listOfNotNull(content), content?.dri)
+    }
 
     private fun PsiDocComment.getDescription(): Description? {
-        val nonEmptyDescriptionElements = descriptionElements.filter { it.text.trim().isNotEmpty() }
-        val convertedDescriptionElements = convertJavadocElements(nonEmptyDescriptionElements)
-        if (convertedDescriptionElements.isNotEmpty()) {
-            return Description(P(convertedDescriptionElements))
+        val nonEmptyDescriptionElements = descriptionElements.filter { it.text.isNotBlank() }
+        return convertJavadocElements(nonEmptyDescriptionElements).takeIf { it.isNotEmpty() }?.let {
+            Description(wrapTagIfNecessary(it))
         }
-
-        return null
     }
 
-    private fun convertJavadocElements(elements: Iterable<PsiElement>): List<DocTag> =
-        elements.mapNotNull {
-            when (it) {
-                is PsiReference -> convertJavadocElements(it.children.toList())
-                is PsiInlineDocTag -> listOfNotNull(convertInlineDocTag(it))
-                is PsiDocParamRef -> listOfNotNull(it.toDocumentationLink())
-                is PsiDocTagValue,
-                is LeafPsiElement -> Jsoup.parse(it.text.trim()).body().childNodes().mapNotNull(::convertHtmlNode)
-                else -> null
+    private inner class Parse : (Iterable<PsiElement>, Boolean) -> List<DocTag> {
+        val driMap = mutableMapOf<String, DRI>()
+
+        private fun PsiElement.stringify(): String? = when (this) {
+            is PsiReference -> children.joinToString("") { it.stringify().orEmpty() }
+            is PsiInlineDocTag -> convertInlineDocTag(this)
+            is PsiDocParamRef -> toDocumentationLinkString()
+            is PsiDocTagValue,
+            is LeafPsiElement -> (if ((prevSibling as? PsiDocToken)?.isLeadingAsterisk() == true) text?.trim() else text)?.takeUnless { it.isBlank() }
+            else -> null
+        }
+
+        private fun PsiElement.toDocumentationLinkString(
+            labelElement: PsiElement? = null
+        ): String? =
+            reference?.resolve()?.let {
+                if (it !is PsiParameter) {
+                    val dri = DRI.from(it)
+                    driMap[dri.toString()] = dri
+                    val label = labelElement ?: defaultLabel()
+                    """<a data-dri=$dri>${label.text}</a>"""
```

Comment:
I don't know if the assumption that the output format will always be able to render html is correct. This is basically coupling the JavadocParser with html-based output